x/cron/keeper/grpc_query_schedule.go

```diff
@@ -0,0 +1,56 @@
+package keeper
+
+import (
+	"context"
+
+	"github.com/cosmos/cosmos-sdk/store/prefix"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/query"
+	"github.com/neutron-org/neutron/x/cron/types"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+func (k Keeper) ScheduleAll(c context.Context, req *types.QueryAllScheduleRequest) (*types.QueryAllScheduleResponse, error) {
+	if req == nil {
+		return nil, status.Error(codes.InvalidArgument, "invalid request")
+	}
+
+	var schedules []types.Schedule
+	ctx := sdk.UnwrapSDKContext(c)
+
+	store := ctx.KVStore(k.storeKey)
+	scheduleStore := prefix.NewStore(store, types.ScheduleKey)
+
+	pageRes, err := query.Paginate(scheduleStore, req.Pagination, func(key []byte, value []byte) error {
+		var schedule types.Schedule
+		if err := k.cdc.Unmarshal(value, &schedule); err != nil {
```

Comment:
There are a lot of places where you use MustUnmarshal/MustMarshal. I think we need to make a choose

Reply:
This was code generated. But I can fix